Tasks/DockerV2/utils.ts

```diff
@@ -24,4 +24,26 @@ export function writeTaskOutput(commandName: string, output: string): string {
     }
     
     return taskOutputPath;
+}
+
+export function getDelimitedInput(inputVal: string, delimters: string[]): string[] {
+    if (!inputVal) {
+        return [];
+    }
+
+    let regex = undefined;
+    if (delimters.length > 1) {
+        regex = new RegExp(delimters.join("|"));
+    } else {
+        regex = new RegExp(delimters[0]);
+    }
+
+    let result: string[] = [];
+    inputVal.split(regex).forEach((x: string) => {
```

Comment:
Doesn't split return an array?

Reply:
Split does return an array but is any tag input has "\n\n" or ",," then it will return array with one extra element as empty string. This empty string needs to be handled separately when appending tags. Hence the extra processing. I have tested it as well.

Reply:
That's still okay. You simply just trim and validate if the tag is non-empty and go forward. This function seems like reinventing the wheel.